### PR TITLE
Git tab feature

### PR DESCRIPTION
**Pull request by mashuDuek**
Add Git tab. 
Style the links bar to fit new tab, and not overlap when resizing screen. 
Add setup instructions to README.

### CODE REVIEW

File: assets/src/entry.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ const smile = () => {
 };
 
 const insertDeskHash = () => {
+  // debugger
```
Comment (OscarAlvarez8830) [inline assets/src/entry.jsx]: Remove debuggers and comments.

File: assets/src/components/app/links/view.jsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ const linksSection = ({ title, links }, idx) => {
   return (
     <section className="links-subs"
       key={ `links-section-${idx}` }>
-      <h2 className="title">{ title }</h2>
+      <h2 id="tab-title" className="title">{ title }</h2>
```
Comment (OscarAlvarez8830) [inline assets/src/components/app/links/view.jsx]: This is referring to multiple elements (it gets mapped to an array). Should be a class, not an id.

File: assets/src/scss/links.scss [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
   min-height: 55px;
-  width: 94%;
   display: flex;
   flex-direction: row;
   border: 2px solid #ccc;
   margin: auto;
   cursor: pointer;
   overflow: visible;
 
+
```
Comment (OscarAlvarez8830) [inline assets/src/scss/links.scss]: Remove extraneous newlines.

File: assets/src/scss/links.scss [hunk 4]
```diff
@@ -22,11 +26,17 @@
...
-      font-weight: 700;
+      font-weight: 300;
       color: darken(#ccc, 50%);
       border-bottom: 4px solid gray;
+      display: flex;
+      justify-content: space-around;
+      margin-left: -10px;
+      margin-right: -10px;
+
+
```
Comment (OscarAlvarez8830) [inline assets/src/scss/links.scss]: Remove extraneous newlines. Also, be sure your styling applies on multiple screen sizes.

File: assets/src/components/app/links/view.jsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ const linksSection = ({ title, links }, idx) => {
   return (
     <section className="links-subs"
       key={ `links-section-${idx}` }>
-      <h2 className="title">{ title }</h2>
+      <h2 className="title tab-title">{ title }</h2>
```
Comment (BCrawfordScott) [inline assets/src/components/app/links/view.jsx]: Remove unused className

**Comment by OhBall** [comment]
The styling on the "Change Commit Authorship" list item looks strange. It's the only two line list item, and looks like there are two separate items: 'Change Commit' and 'Authorship'. I'm not sure what a better solution would be that would keep the styling consistent, though.

**Pull request merged**